src/Microsoft.Dnx.Tooling/ClearCache/ClearCacheCommand.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal class ClearCacheCommand
+    {
+        public Reports Reports { get; }
+        public string HttpCacheDirectory { get; }
+        public ClearCacheCommand(Reports reports, string httpCacheDirectory)
+        {
+            Reports = reports;
+            HttpCacheDirectory = httpCacheDirectory;
+        }
+        
+        public int Execute()
+        {
+            if(Directory.Exists(HttpCacheDirectory))
+            {
+                Reports.Information.WriteLine($"Clearing cache directory {HttpCacheDirectory}");
+
+                Exception exception = null;
+
+                for (var i = 0; i < 3; i++)
+                {
+                    try
+                    {
+                        DeleteDirectoryRecursively(HttpCacheDirectory, false);
+                        Reports.Information.WriteLine("Cache cleared.");
+                        return 0;
+                    }
+                    catch (Exception e)
+                    {
+                        exception = e;
```

Comment:
Mainly temporary open handles (antivirus?) could raise `IOException`.  Much more rarely, if two parallel `dnu clearcache` commands are running at the same time, we might get a `DirectoryNotFoundException` or `FileNotFoundException`. These would go away on the retry.

We hit similar issues with TestUtils in the past - file cleanup code there retries 3 times and it seemed to work well. I'm open to other suggestions.

Good catch with the missing delay. Half a second or so sound good?